sklearn/linear_model/_logistic.py

```diff
@@ -1140,11 +1162,15 @@ class LogisticRegression(LinearClassifierMixin,
         For the liblinear and lbfgs solvers set verbose to any positive
         number for verbosity.
 
+        |ResourcesParameter|
+
     warm_start : bool, default=False
         When set to True, reuse the solution of the previous call to fit as
         initialization, otherwise, just erase the previous solution.
         Useless for liblinear solver. See :term:`the Glossary <warm_start>`.
 
+        |ResourcesParameter|
```

Comment:
I think this should be double tagged as control as well. So this drastically changes behavior of the estimator if you are calling it multiple times.